src/graphics/program-lib/chunks/cubeMapRotate.frag

```diff
@@ -0,0 +1,19 @@
+#ifdef CUBEMAPROT
+uniform mat3 cubeMapRotationMatrix;
+#endif
+
+vec3 cubeMapRotate0(vec3 refDir) {
+#ifdef CUBEMAPROT
+    return refDir*cubeMapRotationMatrix;
+#else
+    return refDir * vec3(-1, 1, 1);
```

Comment:
maybe return vec3(-refDir.x, refDir.yz);

Reply:
yes - I will change it

Reply:
changed - thanks!